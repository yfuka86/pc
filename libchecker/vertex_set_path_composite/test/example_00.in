5 5
1 2
3 4
5 6
7 8
9 10
0 1
1 2
2 3
2 4
1 0 3 11
1 2 4 12
0 2 13 14
1 0 4 15
1 2 2 16
