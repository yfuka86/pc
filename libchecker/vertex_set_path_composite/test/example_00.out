1555
604
6571
222
